query-engine/query-engine-node-api/src/engine.rs

```diff
@@ -173,29 +169,31 @@ impl QueryEngine {
 
         Ok(Self {
             inner: Arc::new(RwLock::new(Inner::Builder(builder))),
-            log_level,
-            log_queries,
-            log_callback,
+            logger: Self::create_log_dispatch(log_queries, log_level, log_callback),
```

Comment:
Maybe `logger::create_log_dispatch(..)`?

Reply:
Good idea. I've changed it to that.